system/system_windows.go

```diff
@@ -301,6 +301,8 @@ func InstallableArchs() ([]string, error) {
 		return []string{"noarch", "x86_32", "x86_64"}, nil
 	case runtime.GOARCH == "arm":
 		return []string{"noarch", "arm"}, nil
+	case runtime.GOARCH == "arm64":
+		return []string{"noarch", "x86_32", "x86_64", "arm64"}, nil
```

Comment:
Just want to confirm that arm64 supports running x86 binaries, since this appears different from arm above.

Reply:
According to Microsoft's Prism emulator documentation[1], ARM64 is capable of running both x86 and x86_64 binaries. Prism is not available on Windows ARM32 platforms.

[1] https://learn.microsoft.com/en-us/windows/arm/apps-on-arm-x86-emulation